### ISSUE TITLE
Create `tractive {command}` for RevMap and migration tasks

### ISSUE THREAD

**Issue by ronaldtse** [ietf-ribose/tractive#41]
```sh
$ tractive generate-revmap reponame.fo reponame-revmap.txt
$ tractive migrate -c config.yml
```

And update documentation please. Thanks!

**Comment by ronaldtse** [comment]
In `config.yml`, we should use:

https://github.com/ietf-ribose/tractive/blob/a166bdbcd8c326c5318242eca9a35a91ef27fd53/config.example.yaml#L22-L23

To configure the revmap path.

**Comment by ronaldtse** [comment]
Closed by #44 #42
